SofascoreAcademy/SofascoreAcademy/LeagueView.swift

```diff
@@ -11,70 +11,57 @@ import SofaAcademic
 
 class LeagueView: BaseView {
```

Comment:
Malo ovaj naming sada vise ne odgovara iz razloga sto ce ovaj view sada pokazivati vise razlicitih liga. 

Mozes preimenovati, ali obzirom da sada ovaj view sadrzi samo tableview (i ne znam sto je s matchViews), jel ti treba dodatan view u hijerarhiji ili moze tableview direkt u viewcontroller?

Reply:
Razmislja sam kao da nemam previse nabacanog koda u ViewControlleru, pa da sve što mogu odvojizi odvojim u komponente, preimenujem onda ovo u leaguesView ili leagueTableView